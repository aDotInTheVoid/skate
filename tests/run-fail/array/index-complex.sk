fn main() {
    print [];
    let x = [[], [], []];
    let x = [[x], x, [[x]]];
    x[x];
}
