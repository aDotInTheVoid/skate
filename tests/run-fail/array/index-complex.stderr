error: Expected `int`, got `array`
  ┌─ $DD/tests/run-fail/array/index-complex.sk:5:7
  │
5 │     x[x];
  │       ^ Evaluated to `[[[[], [], []]], [[], [], []], [[[[], [], []]]]]`

